Tests/DatadogTests/Datadog/RUMMonitorTests.swift

```diff
@@ -96,8 +96,9 @@ class RUMMonitorTests: XCTestCase {
 
         let monitor = RUMMonitor.initialize(rumApplicationID: "abc-123")
 
+        let mockName = String.mockRandom()
         monitor.startView(viewController: mockView)
-        monitor.registerUserAction(type: .tap)
+        monitor.registerUserAction(type: .tap, name: mockName)
```

Comment:
The `mockName` is ambiguous. Let's either call it `mockUserActionName` or use static `String` directly:
```swift
monitor.registerUserAction(type: .tap, name: "UA name")

// ...

XCTAssertEqual(rumModel.action.target?.name, "UA name")
```